x/ecocredit/basket/keeper/utils_test.go

```diff
@@ -87,6 +89,58 @@ func TestGetBasketBalances(t *testing.T) {
 	require.Equal(t, IDToBalance[2], amtToDeposit)
 }
 
+func FuzzCreditAmountToBasketCoin(f *testing.F) {
```

Comment:
added a fuzz test to ensure we are properly converting credit decimal amounts to basket token integers.